docs_src/manuals/user_manual/execution_model.hrst

```diff
@@ -3,35 +3,31 @@
 Execution Model
 ===============
 
-Stencil operations are executed in a three dimensional :term:`Iteration space`. The first two dimensions of the
+Stencil operations are executed in a three dimensional :term:`Iteration Space`. The first two dimensions of the
 iteration space, usually referred to as `I` and `J` dimensions identify the horizontal dimension. There is no
-prescription on in what order stencil operators in different points of the same `IJ` plane will be executed. Stencil
-operators in the third dimension of the iteration space, usually referred as `K` or vertical dimension, can have
-prescribed order of executions. There are three different ways of execution policies for the `K` dimension:
+prescription for the order of stencil operators in different points of the same `IJ` plane. Stencil operators in the
+third dimension of the iteration space, usually referred as `K` or vertical dimension, can have prescribed order
+of executions. There are three different execution policies for the `K` dimension:
 
 - `forward`: The computation at index `k` in the vertical dimension is executed after index `k - 1` for all points in
   the horizontal plane;
 - `backward`: The computation at index `k` in the vertical dimension is executed after index `k + 1` for all points in
   the horizontal plane;
 - `parallel`: No order is specified and execution can happen concurrently.
 
-More concretely, a multistage is a list of stages (implemented with stencil operators) to be executed with a certain
-execution policy.  A computation combines several multistages, and will execute one multistage after the other.
+More concretely, a multistage is a list of stages, implemented with stencil operators, to be executed with a certain
+execution policy. A computation combines several multistages and is executed one multistage after the other.
```

Comment:
```suggestion
execution policy. A computation combines several multistages and executes one multistage after the other.
```
right?

Reply:
Either way sounds good, but I prefer your way. I'll accept the fix!